### PR TITLE
Add the first ethereum api requests

### LINKED ISSUE #69: Connect to the blockchain using an api

### PR DESCRIPTION
**Pull request by 0xdieman**
Closes #69

### CODE REVIEW

**Comment by 0xdieman** [comment]
Okay this is weird there is literally a 100% coverage here because it is only one function but codeclimate is angry again.

**Comment by 0xdieman** [comment]
Is it because of codeclimate ?

**Comment by 0xdieman** [comment]
Is it because of Russia ?

**Comment by 0xdieman** [comment]
Not trying to be political but we never know

**Comment by Zotchh** [comment]
@0xdieman probably Russia, should it be merged for this week or is it some extra that can wait ? I saw that you already made a PR for this sprint.

**Comment by 0xdieman** [comment]
@Zotchh 
It was just a side task but I would like to merge it, I'll see if it's because of the Futures.

**Comment by ahominal** [comment]
> Is it because of Russia ?

I really dont think so ...
I merged and it passes for me on local

**Comment by 0xdieman** [comment]
There seem to be an issue with codeclimate detecting tests that are not in `androidTest`

File: app/src/androidTest/java/com/sdp/swiftwallet/JavaTest/Web3RequestsTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.math.BigInteger;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@RunWith(JUnit4.class)
+public class Web3RequestsTest {
+    public static final String deadWallet = "0x000000000000000000000000000000000000dEaD"; // can I have the private key pls ?
```
Comment (Zotchh) [inline app/src/androidTest/java/com/sdp/swiftwallet/JavaTest/Web3RequestsTest.java:19]: Maybe just put the constant in uppercase to keep naming convention

**Review by Zotchh — APPROVED** [review]
Nothing to add, what was the problem in the end ? It seems from the commit that you moved the test to androidTest  and it solved it ?

**Pull request merged**